### ISSUE TITLE
Update Algorithm and Examples to current JSON Schema

### ISSUE THREAD

**Issue by Echsecutor** [RalphTro/epcis-event-hash-generator#37]
The JSON Schema has changed and the current list of Examples does not conform to the schema definition
https://github.com/gs1/EPCIS/blob/master/JSON/EPCIS-JSON-Schema-single-event.json

The algorithm (JSON parsing) needs to be adapted to accommodate these changes.

**Comment by dakbhavesh** [comment]
Hi @Echsecutor, Have raised PR #38 to support single event in JSON payload.

**Comment by RalphTro** [comment]
**Update** 
I just checked all .jsonld example files again. In the master branch, only `epcisDocWithSensorComponent.jsonld` does not yet conform with the JSON schema yet. With PR 'examplesWithComponent', this is fixed. In this context, I also adjusted the URI pointing to the context file for all examples (see todays commit in the 'examplesWithComponent' branch.
So, IMO, once PR  'examplesWithComponent' is merged into master, we can close this issue.
